files/fr/web/css/color-scheme/index.html

```diff
@@ -0,0 +1,72 @@
+---
+title: color-scheme
+slug: Web/CSS/color-scheme
+browser-compat: css.properties.color-scheme
+translation_of: Web/CSS/color-scheme
+---
+<div>{{CSSRef}}</div>
+
+<p>La propriété CSS <strong><code>color-scheme</code></strong> permet d'indiquer les jeux de couleurs dans lesquel un élément peut être rendu sans risque.</p>
```

Comment:
typo "lesquel" > "lesquelles"

Reply:
Maybe we should rather use "lesquels" given "jeux" is masculine? 🤔

Reply:
OK for "lesquels"👌

Reply:
Il semble que celle-ci est restée (s à la fin) :x

Reply:
Fixed w/ 4869520bd9d6988ecdc192334ea8e2cf40e7f3a9